docs/USAGE-GUIDE.md

```diff
@@ -1585,6 +1585,178 @@ int s2n_config_add_ticket_crypto_key(struct s2n_config *config, const uint8_t *n
 **s2n_config_add_ticket_crypto_key** adds session ticket key on the server side. It would be ideal to add new keys after every (encrypt_decrypt_key_lifetime_in_nanos/2) nanos because
 this will allow for gradual and linear transition of a key from encrypt-decrypt state to decrypt-only state.
 
+## TLS1.3 Pre-Shared Key (PSK) Related Calls 
+
+### s2n\_external\_psk\_new
+
+```c
+struct s2n_psk* s2n_external_psk_new();
+```
+
+**s2n_external_psk_new** creates a new s2n external PSK object with `S2N_PSK_HMAC_SHA256` as the default PSK hash algorithm. Use `s2n_psk_free` to free the memory allocated to the s2n external PSK object.
```

Comment:
You're repeating the comments from s2n.h.

Consider discussing multiple APIs at once instead of each individually, and giving more information about how they interact and can be combined to actually use PSKs. If I'm a customer who just wants to use PSKs, how do I know which of these APIs I need to care about when? Imagine your customer as someone who wants to use pre-shared keys in S2N but hasn't necessarily read the code yet and doesn't necessarily know all the lingo / terms.